### DISCUSSION TITLE
ILONDA Automatic fish feeder

### DISCUSSION THREAD

**Discussion by Trx899** [RavenSystem/esp-homekit-devices#1670] [MEPLHAA Scripts Collection]
-ILONDA automatic fish feeder.
-https://www.amazon.com/iLONDA-Automatic-Adjustable-Compatible-Shockproof/dp/B07LFZ92ZR?th=1
-v.9.5.1
-automatic feed function (1 rotation cicle).
-{"c": {"l":5},"tz":"CET-1CEST-2,M3.5.0/2,M10.5.0/3","tt":[ [11,8,30,-1,1]],"11":{"m":[[1,1]]},"b":[{"g":2,"t":5}],"a":[{"t": 1,"d": 1,"0":{"r":[{"g":14}]},"1":{"r":[{"g":14,"v":1,"i":1}]},"b":[{"g":4,"p":0,"i":1,"t":0}]}]}

Automatic feed function with time table actions to avoid automations, manual function with embedded push button on the device.
Led-GPIO5
Push button-GPIO4
Hall effect switch to stop motion-GPIO12
Relay (Motor)-GPIO14
TYWE3S chip, needs to be flasehd with cables.

**Comment by pawelosin** [comment]
Hi, I am trying to use my json in wemos d1 mini timetable actions, but no luck. Here is my json: 
{"c": {"l":2},"ntp":"pl.pool.ntp.org","tz":"CET-1CEST-2,M3.5.0/2,M10.5.0/3","tt":[[11,6,50]],"11":{"m":[[1,1]]},"b":[{"g":0,"t":5}],"a":[{"t": 1,"0":{"r":[{"g":14}]},"1":{"r":[{"g":14,"v":1}]},"b":[{"g":4,"p":0,"t":0}]}]}.
If I’m right every day at 6:50 my first accessory should be on but isn’t. Can You help me?

**Reply by RavenSystem** [reply]
A brace is wrong. 

{"c": {"l":2,"ntp":"pl.pool.ntp.org","tz":"CET-1CEST-2,M3.5.0/2,M10.5.0/3","tt":[[11,6,50]],"11":{"m":[[1,1]]},"b":[{"g":0,"t":5}]},"a":[{"t": 1,"0":{"r":[{"g":14}]},"1":{"r":[{"g":14,"v":1}]},"b":[{"g":4,"p":0,"t":0}]}]}

**Reply by pawelosin** [reply]
Of course, my mistake, thank you RavenSystem.
